bundles/org.openhab.binding.nest/README.md

```diff
@@ -1,48 +1,36 @@
-| Nest Account (SDM, WWN)                 | An account for using the Nest (SDM/WWN) REST API                                                                                               | sdm_account    | wwn_account        |
-| Nest Cam (Indoor, IQ, Outdoor), Dropcam | A Nest Cam registered with your account                                                                                                        | sdm_camera     | wwn_camera         |
-| Nest Hello Doorbell                     | A Nest Doorbell registered with your account                                                                                                   | sdm_doorbell   | wwn_camera         |
-| Nest Hub (Max)                          | A Nest Display registered with your account                                                                                                    | sdm_display    | wwn_camera         |
-| Nest Protect                            | The smoke detector/Nest Protect for the account                                                                                                |                | wwn_smoke_detector |
-| Nest Thermostat (E)                     | A Thermostat to control the various aspects of the house's HVAC system                                                                         | sdm_thermostat | wwn_thermostat     |
-| Structure                               | The Nest structure defines the house the account has setup on Nest. You will only have more than one structure if you have more than one house |                | wwn_structure      |
+| Things                                  | Description                                                            | Thing Type     |
+|-----------------------------------------|------------------------------------------------------------------------|----------------|
+| Nest Account                            | An account for using the Nest SDM REST API                             | sdm_account    |
+| Nest Cam (Indoor, IQ, Outdoor), Dropcam | A Nest Cam registered with your account                                | sdm_camera     |
+| Nest Hello Doorbell                     | A Nest Doorbell registered with your account                           | sdm_doorbell   |
+| Nest Hub (Max)                          | A Nest Display registered with your account                            | sdm_display    |
+| Nest Thermostat (E)                     | A Thermostat to control the various aspects of the house's HVAC system | sdm_thermostat |
 
 The SDM API currently does not support Nest Protect devices.
-There are no structure Things when using the SDM API, because the SDM API does not support setting the Home/Away status like the WWN API does.
+There are no structure Things when using the SDM API, because the SDM API does not support setting the Home/Away status.
```

Comment:
I'm not sure if this will be a confusing note when reading this in the future - I'm wondering if the sentence should be removed entirely to keep readers from wondering what a "structure" Thing is?

Reply:
I've removed it!